### ISSUE TITLE
Errors in hypercall.h API documentation

### ISSUE THREAD

**Issue by dbkinder** [projectacrn/acrn-hypervisor#166]
Two problems:

1. doxygen comments desribing `hcall_get_cpu_pm_state` are wrong (looks like a copy/paste from
   another function).  Needs a correct description of what it does, and description of each parameter. 
   Also, delete the blank line after the comment and the function prototype.  Here's how it currently 
   appears:

```
/**
 * @brief Switch VCPU state between Normal/Secure World.
 *
 * @param vcpu Pointer to VCPU data structure
 *
 * @return 0 on success, non-zero on error.
 */

int64_t hcall_get_cpu_pm_state(struct vm *vm, uint64_t cmd, uint64_t param);
```

2. In the next function ( `hcall_world_switch` ), the vcpu parameter doc needs to be spelled correctly (change `@param VCPU` to `@param vcpu`  )

**Comment by jren1** [comment]
Jason, Could you please look at that issue?

**Comment by dbkinder** [comment]
@JasonChenCJ  @jren1  Can you fix the doxygen comments please.

**Comment by dbkinder** [comment]
fixed by #236 and #247
